### ISSUE TITLE
[Bug] certain manga fail to update due to json parse error

### ISSUE THREAD

**Issue by jaredanson** [CarlosEsco/Neko#218]
### Device information
* Tachiyomi version: 2.0.13
* Android phone & version: Android 10 Pixel 2XL

## Steps to reproduce
1. I have my manga auto update once every 12 hours.
2. it will fail on two manga (see attached pic)

### Expected behavior
it should update those manga

### Actual behavior
it appears that all except those two get updated.

### Other details
picture of log output

![Screenshot_20200526-151314](https://user-images.githubusercontent.com/8980095/82947057-f43c8200-9f8e-11ea-9f1a-51e75370a03f.png)

**Comment by CarlosEsco** [comment]
rezero is fixed for next release.
the other manga if you manually refresh in the manga info page does it load?  or show error?

**Comment by jaredanson** [comment]
@CarlosEsco if i go to the manga info page and manually refresh I do not get an error. It is only during the scheduled update or if i do a manual refresh for all manga.

I just did a manual refresh, and the notification says `2 updates failed` like last time. But this time it is rezero and another manga. I checked the logs and it is the same log as my screen shot except the new manga title replaced the one in the screen shot. But it is the same `unexpected json token at offset 0` error.

My guess is that this is just a rezero issue, and whichever manga is checked immediately after that one is showing up in the logs and notification as an error as well.
